pages/about/projects/index.vue

```diff
@@ -281,35 +281,31 @@ export default {
             items: [],
             total: 0
           }
-          this.fetchConsortiaStyle(this.consortiaType.id)
           this.fetchResults()
         } 
-      },
-      immediate: true
+      }
     },
     '$route.query.search': {
       handler: function () {
         this.searchQuery = this.$route.query.search
         this.fetchResults()
-      },
-      immediate: true
+      }
     },
     '$route.query.projectsSort': {
       handler: function (option) {
         this.fetchResults()
-      },
-      immediate: true
+      }
     },
     '$route.query.selectedProjectsAnatomicalFocusIds': {
       handler: function (option) {
         this.fetchResults()
-      },
-      immediate: true
+      }
     },
   },
 
   beforeMount: function () {
     this.windowWidth = window.innerWidth
+    this.fetchConsortiaStyle(this.consortiaType.id)
```

Comment:
you cannot move this here. We need to fetch the consortia style when a user switches tabs. It is broken if this is moved to here since this is only ran on initial page load and page refresh

Reply:
sorry for that. it should be working properly now...

Reply:
I don't see any changes....

Reply:
The styling works properly again, but the number of results is still wrong